src/main/java/Duke.java

```diff
@@ -1,10 +1,233 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    private static Task[] taskList = new Task[100];
+    private static int count = 0;
+
+    // These are the prefix strings to define the data type of a command parameter
+    private static final String COMMAND_TODO_WORD = "todo";
```

Comment:
Maybe you can consider using Enum for these constants.

Reply:
Agree with this comment.